### DISCUSSION TITLE
Dynamic Multi-domain Support?

### DISCUSSION THREAD

**Discussion by FractalMind** [evertramos/nginx-proxy-automation#327] [Q&A]
Good day, you have an awesome project going on here.

We are using a **multi-tenant website builder**. Meaning: multiple domains can be linked to ONE web server and it will display the correct webpage according to the domain detected.

I'm looking for an automatic SSL generation system that can route an infinite amount of websites to ONE container that has that **multi-tenant webserver**.

```
Eg: 
exemple1.com, exemple2.com, exemple3.com, exemple4.com => (container01 with multi-tenant server) 
```

Would it be possible to use your system to route an infinite amount of domains to ONE container?

Thank you for your time.

**Comment by evertramos** [comment]
Hello @FractalMind!

If I understood it correctly, you have multiple domains and want them to redirect all to one single container, correct? In order to do that you must add all domains, comma sepparated, in the VIRTUAL_HOST and LETSENCRYPT envrironment directive. 

You algo can add a new one and just run a compose restart and your new url will be added to your nginx proxy.

Give a try on that!
